Telemetry/CorePing.swift

```diff
@@ -64,6 +69,13 @@ class CorePing: TelemetryPing {
 
         let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
 
+        let usageCount = UsageTelemetry.getCount(prefs)
+        let usageTime = UsageTelemetry.getTime(prefs)
+        UsageTelemetry.reset(prefs)
```

Comment:
For all practical scenarios, correct. We record the usage only when entering the background, and we reset only when we send the core ping (which is on the first startup and for each background, after we record). It's theoretically possible for these to overlap if the user manages to background the app very quickly after starting it for the first time, or if they background multiple times very quickly and have queued core pings fire when resetting, but I'm not convinced this is a real concern.